configure.ac

```diff
@@ -26,7 +26,7 @@ AC_TYPE_UINT8_T
 AC_CHECK_FUNCS([malloc memmove memset strchr strtol])
 
 AX_CFLAGS_WARN_ALL
-AX_COMPILER_FLAGS(,,,[-Wpedantic -Wall -Wextra -Wcast-align -Wcast-qual\
+AX_COMPILER_FLAGS([-Wpedantic -Wall -Wextra -Wcast-align -Wcast-qual\
```

Comment:
Why this change? Per the AX_COMPILER_FLAGS documentation, I'm purposefully setting the EXTRA-BASE-CFLAGS parameter for the following behavior (quoting the documentation):

    The set of base and enabled flags can be augmented using the EXTRA-*-CFLAGS and EXTRA-*-LDFLAGS variables, which are tested and appended to the output variable if –enable-compile-warnings is not "no".